lib/govuk_message_queue_consumer/test_helpers/mock_message.rb

```diff
@@ -6,9 +6,10 @@ class MockMessage < Message
     alias :discarded? :discarded
     alias :retried? :retried
 
-    def initialize(payload = {}, options = {})
+    def initialize(payload = {}, options = {}, delivery_info = {})
```

Comment:
The `options` arg can be replaced by `headers` (there are no other options to specify).